eventuate-tram-view-support-e2e-tests/src/test/java/io/eventuate/tram/viewsupport/e2e/tests/TestExportSnapshots.java

```diff
@@ -54,11 +53,11 @@ public void testThatViewsMatchOriginalEntitiesAfterExport() {
 
     domainEntityDomainSnapshotExportService.exportSnapshots();
 
-    TopicPartitionOffset[] topicPartitionOffsets = restTemplate.postForObject(String.format("http://localhost:%s/export/test-domain-entity", port), null, TopicPartitionOffset[].class);
+    TopicPartitionOffsetMessageId[] topicPartitionOffsetMessageIds = restTemplate.postForObject(String.format("http://localhost:%s/export/test-domain-entity", port), null, TopicPartitionOffsetMessageId[].class);
```

Comment:
Having class names (and variable names) that are a concatenation of the classes fields doesn't seem like a winning strategy. Add a new field => change the name everywhere....

Reply:
I will improve it